powerstore/resource_filesystem_snapshot.go

```diff
@@ -33,6 +31,16 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
 	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
 	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"terraform-provider-powerstore/client"
+	"terraform-provider-powerstore/models"
+)
+
+const (
+	// DefaultExpirationTimestamp is the default expiration timestamp when not specified
```

Comment:
Update the documentation to use this value for setting unlimited retention after snapshot creation